pkg/kubehound/models/converter/graph.go

```diff
@@ -27,7 +27,7 @@ func (c *GraphConverter) Container(input *store.Container, parent *store.Pod) (*
 		App:         input.Ownership.Application,
 		Team:        input.Ownership.Team,
 		Service:     input.Ownership.Service,
-		Namespace:   input.Inherited.Namespace,
+		Namespace:   parent.K8.Namespace,
```

Comment:
why this change?

Reply:
It changed because it was a error in the container config. It should inherit from the pod. Otherwise the namespace is not set correctly. I stubbled it during the unit tests when I added the vault-pod which was in a different namespace.
This should have been on the previous PR, but I mixed it.

Reply:
Fixed it in the generator instead.